### ISSUE TITLE
Inconsistent capitalisation of classnames for inputs in auto-generated code.

### ISSUE THREAD

**Issue by tom-homewood-dyson** [awslabs/aws-mobile-appsync-sdk-android#122]
**Describe the bug**
Capitalisation of class names generated by InputTypeSpecBuilder is not consistent with the names used in import statements in other classes.

**To Reproduce**
1. Create an Appsync API with a schema containing an input as follows:
```
input foo_bar {
	timestamp: Float!
	value: Float!
}
```
2. Add a mutation to the schema as follows:
```
type Mutation {
	updateStuff(foo: foo_bar): Stuff
	...
        ...
}
```
3. Attach the API to the project and build. Amplify Codegen will create files for the input (Foo_bar) and the mutation (UpdateStuffMutation).

4. The build will fail due to the UpdateStuffMutation incorrectly referencing the input as follows:
```
import type.foo_bar;
```

**Which AWS service(s) are affected?**
Appsync / Amplify codegen (not sure where the actual bug is)

**Expected behavior**
The import statement in the mutation class file is *capitalised to match the name of the input class file, as below*:
```
import type.Foo_bar;
```

**Environment Information (please complete the following information):**
 - AWS Android SDK Version: [e.g. 2.7.6].
 - Device: N/A (app will not compile).
 - Android Version: App will not compile but compileSdkVersion = 28
 - Specific to simulators: No

(I'm aware that using underscore in input and type names is probably not the best idea, but nevertheless this inconsistent capitalisation is still a bug).

**Comment by cbommas** [comment]
@tom-homewood-dyson 

Thanks for bringing this issue to my attention. I am looking into this and will post an update here once I have more information.

**Comment by desokroshan** [comment]
This should have been fixed with version 2.8.3 of the SDK. Could you please upgrade and confirm @tom-homewood-dyson.
